packages/application-config/src/formatters.ts

```diff
@@ -44,15 +45,7 @@ const formatEntryPointUriPathToResourceAccessKey = (
     .join('_')
     // Each word is split by a hyphen.
     .split('-')
-    .map((word, i) => {
-      // Regex below checking if the character is numeric.
-      // If the word after the hyphen is numeric, replace the hyphen with a forward slash.
-      // If not, omit the hyphen and uppercase the first character
-      if (i > 0 && /^-?\d+$/.test(word[0])) {
-        return `/${word}`;
-      }
-      return upperFirst(word);
-    })
```

Comment:
This formatting of digits was removed (as discussed)

Reply:
Wouldn't this be a breaking change?
If we have an already installed application with the entry point `avengers-01`, the resources access keys used to create the permission resource name which would be stored in the database would be `viewAvengers/01` and `manageAvengers/01`.
After updating the `app-kit`, the permission resource name calculated in the application at runtime would be `viewAvengers01` and that would not match the one in the database.

Maybe I'm missing something, but I wanted to double check.

Reply:
Hmm 🤔, you are right, thanks. It's possible I misunderstood something... 
I guess I can have another look if this digit formatting is somehow transferrable to types inference.

Reply:
Right, my bad, sorry for that